jekyll/_cci2/migrating-from-1-2-experimental-translator.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: classic-docs
+title: "Experimental Translator from 1.0 to 2.0 config"
+short-title: "Experimental Translator for CircleCI 1.0 to 2.0"
+description: "Instructions for using the experimental Translator to translate CircleCI 1.0 configuration to the 2.0 format."
+categories: [migration]
+hide: true
+order: 15
+---
+
+This document provides instructions for using an experimental Translator that generates a CircleCI 2.0 configuration based on an existing 1.0 project.
+
+* Contents
+{:toc}
+
+## Important Caveat:
+This Translator is designed to produce working configuration, but not necessarily optimal configuration. The configuration generated is commented to help you understand the inferred commands that the 1.0 build engine executed. _For production use, we strongly recommend reading and manually creating your 2.0 configuration using our [documentation on migrating from 1.0 to 2.0]({{ site.baseurl }}/2.0/migrating-from-1-2/)_.
+
+## Translator Overview
+The configuration Translator is an experimental project to help with migrating from CircleCI 1.0 to 2.0. 
+
+The approach in CircleCI 2.0 differs in a few significant ways that make an automated migration complicated:
+1. **CircleCI 1.0 has "magic" inferred commands, whereas 2.0 is wholly deterministic.** In CircleCI 1.0 our Inference Engine runs implicit commands based on the contents of your repository in many cases. For instance, unless overridden, CircleCI 1.0 would detect the presence of rspec tests and run them automatically with no intervention. In 2.0 our philosophy is to run no "magic" commands, preferring instead to have all configuration be transparent and deterministic for developers. 
+2. **CircleCI 1.0 has an opinionated lifecycle, whereas 2.0 is built around arbitrary jobs orchestrated with workflows.** In CircleCI 1.0 your build consists of specific stages that are baked into the system, and the configuration semantics are structured around those stages. In CircleCI 2.0 we allow for arbitrary jobs that are orchestrated with Workflows, so the idiom for how to structure the various tasks of your build is quite different and more powerful in 2.0.
+
+## Translator Limitations
+### Supported Languages
+The Translator currently attempts to support transfer of inferred commands for the following languages (with some limitations for each):
+* Ruby
+* PHP
+* Node.js
+* iOS (partial)
```

Comment:
Let's call out here that "partial" refers to no automatic code-signing.

`* iOS (partial - 1.0 code signing is not supported)`